src/JestManager.ts

```diff
@@ -1,8 +1,10 @@
 import { JestTotalResults, Options, Runner, TestReconciler } from "jest-editor-support";
 import { WorkspaceFolder } from "vscode";
-import { createRunner } from './JestSettings';
-import { ProjectConfig } from './repo';
+import { createRunner } from "./JestSettings";
```

Comment:
True.  I'm not sure why I'm getting the diff on the formatting.  I suspect that other contributors may have different configurations.  But it may just be a legacy thing.  I'm not too fussed about separate PRs at this stage, but I do take your point.  I did consider whether to include those changes when committing.

Reply:
Even tossing them in a separate commit can be helpful.  I'll do that on my own stuff so that when I'm looking at the changes I'm about to commit for the actual functionality, it's easier to do a self-code-review of them